recipes/mender-shell/debian-master/control

```diff
@@ -0,0 +1,12 @@
+Source: mender-shell
+Maintainer: Mender Team <mender@northern.tech>
+Section: misc
+Priority: optional
+Standards-Version: 3.9.2
+Build-Depends: debhelper (>= 9)
```

Comment:
why we are not specifying the build depends? as in e.g.: libgio dev packages?

Reply:
Well spotted :+1:

But  we are not using these `Build-Depends` at all in our build system. I'll deal with this when preparing the official Debian recipes.